README.md

```
@@ -0,0 +1,33 @@
+## Usage
+
+
+```js
+import { multiply } from 'react-native-ble-utils';
+
+// ...
+
+const result = await multiply(3, 7);
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)